.github/actions/teardown-rocm/action.yml

```diff
@@ -17,3 +17,7 @@ runs:
     - name: Runner diskspace health check
       uses: ./.github/actions/diskspace-cleanup
       if: always()
+    - name: Chown workspace back to pytorchci (Fallout of PR 122922)
+      if: always()
+      run: |
+        sudo chown -R pytorchci ${GITHUB_WORKSPACE}
```

Comment:
Oh, so this explains the permission issues when the next jobs fail to clean it up.  Could we not hardcode the username here as it's subjected to change and use something like `$USER` instead?